bool lights_on
---
bool success